scenes/game_elements/characters/enemies/guard/components/guard.gd

```diff
@@ -437,35 +431,6 @@ func edit_patrol_path() -> void:
 		editor_interface.edit_node.call_deferred(patrol_path)
 
 
-func _draw() -> void:
-	## Only draw the patrol path debug line if we are in the editor
-	## and the guard node is selected.
-	if Engine.is_editor_hint() and self in _editor_interface().get_selection().get_selected_nodes():
-		var debug_color: Color = Color.RED
-		var line_width: float = 5.0
-		var point_size: float = 15.
-
-		if patrol_path and patrol_path.curve:
-			var curve: Curve2D = patrol_path.curve
-			## Draw the patrol path segments
-			if curve.point_count > 1:
-				for point_idx in curve.point_count - 1:
-					draw_line(
```

Comment:
I think it's good to remove this because it interacts badly with the debug option to have the guard move in the editor: the path is drawn relative to the guard's current position not original position.